shared-mq/pom.xml

```diff
@@ -1,41 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The shared-mq submodule wasn't referenced as a submodule from the parent pom, and there were no dependencies on it in source code.  I removed it for cleanliness.